src/renderer/component/fileRender/view.jsx

```diff
@@ -59,6 +120,20 @@ class FileRender extends React.PureComponent<Props> {
     return viewer || unsupported;
   }
 
+  escapeListener(e) {
+    if (e.keyCode==27){
```

Comment:
I'm not able to enter full screen. Is this added?

Reply:
Not yet, a lot of work needs to be done to make it work with Electron sandboxing.
The request from the sandbox goes through, but there are issues with dynamically sizing content that has already request a WebGL context.

Reply:
This is added so if fullscreen happens, the user can always escape.  Nothing is stopping existing apps from requesting it. *It can't be cancelled but it can be exited, Electron team hardcoded the events.